distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -8,6 +8,52 @@ import project.Data.Vector.Vector
 @Builtin_Type
 type Function
 
+    ## Applies the function `self` to the provided argument.
+
+       Arguments:
+       - argument: The argument to apply `self` to.
+
+       ? Piping Blocks to Functions
+         This construction is particularly useful for passing a block as an argument
+         to a function. This means that you can compute more sophisticated values
+         in-line, as shown in the example below.
+
+       > Example
+         Applying a function to a block.
+
+             (x -> x + 1) <|
+                y = 1 ^ 3
+                3 + y
+    <| : Any -> Any
+    <| self ~argument = @Builtin_Method "Function.<|"
+
+    ## Composes two functions together, for `f << g` creating the function
+       composition `f ∘ g` (equivalent to `x -> f (g x)`).
+
+       Arguments:
+       - that: The function to compose with `self`.
+
+       > Example
+         Multiply by 2 and then add 1 as a function applied to 2.
+
+             (+1 << *2) 2
+    << : (Any -> Any) -> (Any -> Any)
```

Comment:
There are four failures:
- [Table.join](https://github.com/enso-org/enso/issues/6221#issuecomment-1519398078)
- [Any_Spec](https://github.com/enso-org/enso/actions/runs/4782616196/jobs/8502113085?pr=6384#step:10:7228) - `should be able to be applied to an argument using <|`
- [should be able to be composed backward using <<](https://github.com/enso-org/enso/actions/runs/4782616196/jobs/8502113085?pr=6384#step:10:8386)
- [stack traces](https://github.com/enso-org/enso/actions/runs/4782616196/jobs/8502113085?pr=6384#step:10:9339) `No_Such_Method.Error UnresolvedSymbol<source_location> UnresolvedSymbol<>>>`

Reply:
All the failures go away with f03aba6 and (hopefully also) 5ffa216.